client/src/services/auth.serivce.ts

```diff
@@ -0,0 +1,16 @@
+import axios from "axios";
+import { IRegisterRequest } from "../DTOs/register-request-dto";
+import { IRegisterResponse } from "../DTOs/register-response-dto";
+
+function register(body: IRegisterRequest): Promise<any> {
+  return axios
+    .post<IRegisterRequest, IRegisterResponse>("http://localhost:5000/api/auth/register", body, {
```

Comment:
let's create config file and use 'http://localhost:5000/api' there.

Here, you will use like: `${config.apiUrl}/auth/register`